blueprints/module-unification-app/files/tests/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
The usage of `overrides` in `.eslintrc.js` should remove the need for these files